snap/snapcraft.yaml

```
@@ -675,6 +675,12 @@ parts:
 
       mkdir -p $SNAPCRAFT_PART_INSTALL/config/security-proxy-setup
       cp $SNAPCRAFT_PART_INSTALL/etc/kong/kong.conf.default $SNAPCRAFT_PART_INSTALL/config/security-proxy-setup/kong.conf
+
+      # this step resolves "package contains external symlinks: usr/local/openresty/bin/openresty lint-snap-v2_external_symlinks"
```

super minor nitpick, but rather than explaining the esoteric error that the review-tools pumps out, could you explain what it is doing to resolve and what the problem is, i.e. something like

```suggestion
      # by default the deb contains an absolute symlink @ /usr/local/foo/bar which points to /foo/bar/something
      # because the review-tools for the snap store do not currently allow using absolute symlinks that point outside of the snap
      # (and are not smart enough to realize there is a layout involved in our usage here), we delete the absolute symlink
      # and re-create it as a relative one instead
```

obviously fix the paths I put there

---

Thanks @anonymouse64 - good point. I have changed the comment and re-pushed my changes